### ISSUE TITLE
No CMAKE_C_COMPILER could be found.

### ISSUE THREAD

**Issue by salocinx** [Phylliida/orbslam-windows#10]
Hi thanks for your instructions. I have Visual Studio 2015 v14, CMake, OpenCV etc installed. When trying "generate" the build in step 1 I get the following errors:

CMake Error at CMakeLists.txt:2 (project):
  No CMAKE_C_COMPILER could be found.

CMake Error at CMakeLists.txt:2 (project):
  No CMAKE_CXX_COMPILER could be found.

What do I need to install additionally?

**Comment by Phylliida** [comment]
In the step:

Press Configure and choose Visual Studio 14 2015 Win64 or Visual Studio 12 2013 Win64

It is possible you selected something other than Visual Studio here. You can try deleting the cmake cache (in File -> Delete Cache) and do this again

**Comment by salocinx** [comment]
Thanks for responding. I meanwhile solved the issue. I simply had to re-install Visual Studio and selecting all the C++ compilers and libraries available. Afterwards, the appropriate C/C++ compiler has been found by CMake.

By the way, I summarized all my experiences while compiling [ORBSLAM2 + OpenCV + OpenNI within this guide](http://nicolas.baumgardt.ch/slam/ORBSLAM2_Install_Guide_Win10.pdf). Hopefully others will benefit :-)

Thanks for your support and your great instructions, without I would not have been able to compile ORBSLAM2.
